logging/CHANGELOG.md

```diff
@@ -4,6 +4,29 @@
 
 [1]: https://pypi.org/project/google-cloud-logging/#history
 
+## 1.4.1
+
+### Implementation Changes
+  - Logging pass in grace period and batch size to workers (#4369)
+  - Unique Writer Identity (#4595, #4704, #4706, #4708)
+  - Making Sink.filter Optional (#4699)
+  - Capture Server-generated writerIdentity during LogSink.create (#4707)
+  - Generate GAPIC and Remove GAX (#4759)
```

Comment:
This should be stated as "The underlying client library was re-generated to pick up new features and bugfixes."